integration_tests/pages/apply/beforeYouStartPage.ts

```diff
@@ -14,14 +14,14 @@ export default class BeforeYouStartPage extends Page {
 
   shouldShowApplicationGuidance(): void {
     cy.get('p').contains(
-      'Community Accommodation Service Tier 2 provides housing for people who do not have a suitable address for the term of their licence or Bail Order.',
+      'Use this service to apply for accommodation and support on behalf of someone on Home Detention Curfew (HDC) licence.',
     )
 
     cy.get('h2').contains('Before you start')
-    cy.get('h3').contains('Who is this for?')
+    // cy.get('h3').contains('Who is this for?')
```

Comment:
Could you replace some of this commented out code to test some of the new content? It doesn't have to be comprehensive at all, just check for the `<h3>`, a couple of `<p>` and `<li>` elements.

e.g. `cy.get('h3').contains('h3 text here')`

Let me know if you get stuck :)

Reply:
perfect!